lib/fluent/plugin/out_dynatrace.rb

```diff
@@ -64,7 +64,10 @@ def configure(conf)
         compat_parameters_convert(conf, :inject)
         super
 
-        @uri = URI.parse(@active_gate_url)
+        raise Fluent::ConfigError, 'api_token has not been set' if @api_token.empty?
```

Comment:
```suggestion
        raise Fluent::ConfigError, 'api_token is empty' if @api_token.empty?
```
Would be more precise